torch/_higher_order_ops/invoke_subgraph.py

```diff
@@ -33,6 +33,8 @@
 class InvokeSubgraphHOP(HigherOrderOperator):
     def __init__(self) -> None:
         super().__init__("invoke_subgraph")
+        self.tags = ()
+        self.is_view = False
```

Comment:
Maybe these should be on base HigherOrderOperator?  I'm not sure...

Reply:
The alternative is that we can try to not access these fields in fake tensor's torch_dispatch

Reply:
@anijain2305 do we still need these?